### ISSUE TITLE
city split of so2sat

### ISSUE THREAD

**Issue by shermanlian** [chaudatascience/diverse_channel_vit#2]
Great work on the project! We’re currently reproducing your results and have a couple of questions about the So2Sat-LCZ42 implementation:
1. When you refer to the “city split,” does that mean the the first version (only training/validation split) from the original dataset (https://github.com/zhu-xlab/So2Sat-LCZ42 First version)?
2. Are the “city split” results in your main table computed on the validation set or on the official test set of So2Sat-LCZ42?

Thanks for your help!

**Comment by chaudatascience** [comment]
Hi @shermanlian, 

Thanks for being interested in our work. 

1, It should be the second version, which is the first version plus the test set (352K training images, 24K validation images, and 24K test images). I've just updated the dataloader (`‎datasets/so2sat.py`) to reflect this. Thanks for bringing this up!
2, The test set. The final model was chosen based on the checkpoint with the highest accuracy on the valid set.

Just let me know if you still have questions on this.
Best,
Chau

**Comment by shermanlian** [comment]
Got it thanks!!
